tests/functionnal/Glpi/Inventory/Assets/Computer.php

```diff
@@ -1101,4 +1101,88 @@ public function testAssetTag()
+          </BIOS>
+          <VERSIONCLIENT>FusionInventory-Inventory_v2.4.1-2.fc28</VERSIONCLIENT>
+        </CONTENT>
+        <DEVICEID>glpixps.teclib.infra-2018-10-03-08-42-36</DEVICEID>
+        <QUERY>INVENTORY</QUERY>
+        </REQUEST>";
+
+        $inventory = $this->doInventory($xml_source, true);
+
+        //check created agent itemtype / deviceid / entities_id
+        $agents = $DB->request(['FROM' => \Agent::getTable()]);
+        $this->integer(count($agents))->isIdenticalTo(1);
+        $agent = $agents->current();
+        $this->array($agent)
+            ->string['deviceid']->isIdenticalTo('glpixps.teclib.infra-2018-10-03-08-42-36')
+            ->string['itemtype']->isIdenticalTo('Computer')
+            ->integer['entities_id']->isIdenticalTo(0);
+
+
+        $computers_id = $inventory->getItem()->fields['id'];
+        $this->integer($computers_id)->isGreaterThan(0);
+        //load computer
+        $this->boolean($computer->getFromDB($computers_id))->isTrue();
+        //test entities_id root
+        $this->integer($computer->fields['entities_id'])->isEqualTo(0);
+
+
+        //transer to another entity
+        $doTransfer = \Entity::getUsedConfig('transfers_strategy', $computer->fields['entities_id'], 'transfers_id', 0);
+        $transfer = new \Transfer();
+        $transfer->getFromDB($doTransfer);
+
+        $item_to_transfer = ["Computer" => [$computers_id => $computers_id]];
+        $transfer->moveItems($item_to_transfer, 1, $transfer->fields);
+
+        //reload computer
+        $this->boolean($computer->getFromDB($computers_id))->isTrue();
+        //test computer entities_id
+        $this->integer($computer->fields['entities_id'])->isEqualTo(1);
+        //test agent entities_id
+        $this->integer($computer->fields['entities_id'])->isEqualTo(1);
+
+        //prohibit the transfer from this entity
+        $entity = new \Entity();
+        $entity->getFromDB(1);
+        $this->boolean($entity->update([
+            "id" => $entity->fields['id'],
+            "transfers_id" => 0, //no transfert
+        ]))->isTrue();
+
+        //redo inventory
+        $inventory = $this->doInventory($xml_source, true);
+
+        $computers_id = $inventory->getItem()->fields['id'];
+        $this->integer($computers_id)->isGreaterThan(0);
+        //load computer
+        $this->boolean($computer->getFromDB($computers_id))->isTrue();
+        //test entities_id sub
+        $this->integer($computer->fields['entities_id'])->isEqualTo(1);
+
+
+        //check created agent itemtype / deviceid / entities_id
+        $agents = $DB->request(['FROM' => \Agent::getTable()]);
+        $this->integer(count($agents))->isIdenticalTo(1);
+        $agent = $agents->current();
+        $this->array($agent)
+            ->string['deviceid']->isIdenticalTo('glpixps.teclib.infra-2018-10-03-08-42-36')
+            ->string['itemtype']->isIdenticalTo('Computer')
+            ->integer['entities_id']->isIdenticalTo(1);
```

Comment:
What was happening here before the fix? Was it set to `0`?

Reply:
Yes agent is always on root event f PC stay in right entity

Reply:
:+1: